docs/client-sdk/javascript/tutorials/filter-and-label-conversations-by-id-and-metadata.md

```diff
@@ -0,0 +1,168 @@
+
+* A subject line  
+Filter to display only conversations with a specific subject line, as in email
+
+* An NFT  
+Filter to display only conversations associated with a specific NFT
+
+You can also use conversation IDs and metadata to display conversation labels to provide users with useful information about their conversations.
+
+Use this tutorial to learn how to use conversation IDs and metadata to filter and label conversations in your app.
+
+
+## Set a conversation ID
+
+Set the `conversationId` when creating a conversation.
+
+:::tip
+
+As a best practice, start your `conversationId` with a domain to help avoid collisions between your app and other apps on the XMTP network.
+
+:::
+
+<!--Regarding this tip, do we want to emphasize that the domain should be unique to and/or meaningful in the context of your app? Or is that too prescriptive? Is there a way they can query the network to see which conversationIds are in use?-->
+
+<!--In the Lens tutorial, we call mydomain.xyz/alert the "domain prefix" and then the conversationId appears to be a string composed of the two profiles in the conversation, or does the concatenation of domain prefix and the profile IDs form the conversationId? Maybe I'm taking the format "mydomain.xyz/notif" too literally? Can a conversationId just be any string you want it to be?-->
+
+This example sets `mydomain.xyz/notif` as the `conversationID`, indicating that these conversations are notification messages created by `mydomain.xyz`.
+
+You can then use the `mydomain.xyz/notif` conversation ID to display only notification messages created by your app, for example. <!--made this up - is this a valid use case? I was thinking that you could use this filter to show only notif messages in a notification panel, for example.-->
+
+```js showLineNumbers
+// Start a scoped conversation with ID mydomain.xyz/notif
+const conversation1 = await xmtp.conversations.newConversation(
+  '0x3F11b27F323b62B159D2642964fa27C46C841897',
+  {
+    conversationId: 'mydomain.xyz/notif',
+  }
+)
+```
+
+<!--In the code snippet above - 0x3F11b27F323b62B159D2642964fa27C46C841897 - whose address is this? the sender or the recipient? I was thinking that this would include both addresses? Another part of me is thinking that no address should show at all? For example, if the conversation topic name doesn't include the user addresses - are those addresses obscured everywhere? Not sure if this question makes sense.-->
+
+
+## Set conversation metadata
+
+In addition to setting a conversation ID when creating a conversation, you can set metadata to use as an additional filter.
+
+<!--You can just use metadata without a conversationId if you want, correct? Any guidance or best practices here?-->
+
+This example sets `mydomain.xyz/dm` as the `conversationId`, indicating that these conversations are direct messages. In addition, the example sets a conversation metadata `title` value to `following`.
```

Comment:
I do think we want to make it very clear that you do not need to use conversation IDs to send DMs. It will cause multiple conversations for the same pair of people if they are talking on multiple apps. 

Maybe we don't want this example at all

Reply:
Thank you for this @neekolas! I'm adding info up front making it clear that conversation IDs and metadata are optional. I'm also surfacing that there are downstream impacts of using them.

I will update this example to be about "alerts" instead of DMs - though I am open to another use case!

That said, the last section is specifically about using labels to differentiate conversations in the DM context... so the DM use case is still surfaced there. However, I think it is okay because we are surfacing it along with the solution to the confusing UX that may arise.

Or do you have deeper concerns about calling out the DM use case? I'd love to know your thoughts on this!